lesson03/assignment/index.js

```diff
@@ -0,0 +1,146 @@
+// profiles is an identifier for the array of objects that needs to be added
+const profiles = [
+  {
+    "name": "Paolo Maldini",
+    "jobTitle": "Front End Developer",
+    "company": "Web Weavers",
+    "experience": "3 years",
+    "school": "University of Washington",
+    "major": "Marketing",
+    "email": "paolo@example.com",
+    "linkedInUrl": "paolo.linkedinprofile.com",
+    "codeLanguages": [
+        "HTML", "CSS", "JavaScript", "Node", "Express"
+    ]
+  },
+  {
+    "name": "Barbara Bonansea",
+    "jobTitle": "Software Engineer",
+    "company": "Excellence in the Cloud",
+    "experience": "12 years",
+    "school": "University of Southern California",
+    "major": "Computer Science",
+    "email": "barbara@example.com",
+    "linkedInUrl": "barbara.linkedinprofile.com",
+    "codeLanguages": [
+        "HTML", "JavaScript", "C", "Go"
+    ]
+  },
+  {
+    "name": "Javier Hernandez",
+    "jobTitle": "User Experience Engineer",
+    "company": "Web Sites and More",
+    "experience": "5 years",
+    "school": "Seattle University",
+    "major": "Performing Arts",
+    "email": "javier@example.com",
+    "linkedInUrl": "javier.linkedinprofile.com",
+    "codeLanguages": [
+        "HTML", "CSS"
+    ]
+  },
+  {
+    "name": "Maribel Dominguez",
+    "jobTitle": "Front End Engineer",
+    "company": "Bits and Bytes",
+    "experience": "6 years",
+    "school": "University of Washington",
+    "major": "Mechanical Engineering",
+    "email": "maribel@example.com",
+    "linkedInUrl": "maribel.linkedinprofile.com",
+    "codeLanguages": [
+       "HTML", "CSS", "JavaScript", "React", "Vue", "Redux"
+    ]
+  }
+ ]
+
+
+
+ // Code that requires the DOM be loaded should not be run until the DOM is loaded
+ document.addEventListener("DOMContentLoaded", function(){
+
+
+   //map will loop over each object in the array with a method
+   //end result - new card for each profile (which is an object in the array)
+
+  let cardsHTML = profiles.map(function(x){
+
+    // turn the codeLanguages array into a string with a space between elements
+     let language = x.codeLanguages;
+     let languagesString = language.join(", ")
```

Comment:
Unless you are using `language` more than once, you can probably combine these into one line: `let languageString = x.codeLanguages.join(", ")`